pkg/clickhousegw/clickhousegw.go

```diff
@@ -90,6 +90,8 @@ func (c *ClickHouseGateway) getCreateTableSchemaDDL(isBaseTable bool, zookeeperP
 			agent           IPv6,
 			int_in          String,
 			int_out         String,
+			tos             UInt8,
```

Comment:
Does this automatically do some schema migration? Because the table already exists.

Reply:
It doesn't. We'll need to run it manually unless we implement it here. But how?